src/NuGet.Core/NuGet.Protocol.Core.v3/Utility/LocalFolderUtility.cs

```diff
@@ -1076,5 +1076,18 @@ private static LocalPackageInfo GetPackageV3(string root, string id, string vers
 
             return null;
         }
+
+        private static FileInfo TryCreateFileInfo(string localPath)
```

Comment:
convention for `TryCreate...` methods is to return `bool` and use `out` parameter for the value